include/mgos_crontab.h

```diff
@@ -196,6 +196,17 @@ void mgos_crontab_register_handler(struct mg_str action, mgos_crontab_cb cb,
  */
 time_t mgos_crontab_get_next_invocation(mgos_crontab_job_id_t id, time_t date);
 
+/*
+ * Deactivate all existing cron jobs
+ */
+void mgos_crontab_remove_all();
```

Comment:
void mgos_crontab_remove_all(void);

because [C is weird sometimes](https://softwareengineering.stackexchange.com/a/287002).